packages/headless/src/ssr-next/commerce/controllers/breadcrumb-manager/headless-core-breadcrumb-manager.ssr.ts

```diff
@@ -4,7 +4,7 @@ import type {
 } from '../../../../controllers/commerce/core/breadcrumb-manager/headless-core-breadcrumb-manager.js';
```

Comment:
This comment is totally out of scope, but seeing this makes me realize it would be nice to define path aliases in the headless sub-package 😅